mythril/laser/ethereum/state/memory.py

```diff
@@ -157,6 +157,8 @@ def __setitem__(
                 self[start + i] = cast(List[Union[int, BitVec]], value)[i]
 
         else:
+            if key >= self.__len__():
```

Comment:
Why not use this? Should do the same right?
```suggestion
            if key >= len(self):
```